frontend/src/pages/answer/Answer.jsx

```diff
@@ -111,16 +114,37 @@ export default function Answer() {
     validateAndInitializeMessage(answerResponse);
   }
 
+  /**
+   * Get metadata of answer to see if current user owns it
+   */
+  async function checkIfAnswerOwned() {
+    let accessToken;
+    if (isAuthenticated) {
+      try {
+        accessToken = await getAccessTokenSilently();
+      } catch (err) {
+        displayAlert('error', `Failed to validate user. Error: ${err}`);
```

Comment:
```suggestion
        displayAlert('error', `Failed to authenticate user. Error: ${err}`);
```

Reply:
Updated in 06dfb063189bcbd4f9ce23d3ace08fba1c9301f6